stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/experimental/channel/state/MessagesState.kt

```diff
@@ -0,0 +1,38 @@
+package io.getstream.chat.android.offline.experimental.channel.state
+
+import io.getstream.chat.android.client.models.Message
+import io.getstream.chat.android.offline.ChatDomainImpl
+
+/**
+ * Represents of possible state of messages for [ChannelState].
+ */
+public sealed class MessagesState {
```

Comment:
Same here. Maybe we could add @Experimental and @InternalApi for now

Reply:
Done